tcf_website/tests/test_instructor.py

```diff
@@ -58,3 +59,17 @@ def test_average_difficulty_no_reviews(self):
         self.assertTrue(
             self.instructor.average_rating_for_course(self.course) is
             None)
+
+    def test_instructor_view(self):
+        """Test if context variables are correct in the instructor view."""
+        response = self.client.post(
+            reverse('instructor', args=(self.instructor.id,)))
+        self.assertEqual(response.context[0]['avg_difficulty'], 2.67)
```

Comment:
I'm assuming you manually calculated these expected values from the reviews in `test_utils`? This might be an irrelevant concern, but if the test instances ever changed, we'd have to edit these hardcoded values as well. It looks like other tests in the file (`test_average rating`, etc.) just manually grab the values and run an average calculation within the test, then compare with the output, so that might be the way to go.

Reply:
Fixed in 9fef6f7af4ecccbf86f36aa07dd6a07cc226a80b